packages/react-server-dom-webpack/package.json

```diff
@@ -25,21 +25,27 @@
   "exports": {
     ".": "./index.js",
     "./plugin": "./plugin.js",
-    "./server": "./server.js",
-    "./server.browser": "./server.browser.js",
-    "./server.node": "./server.node.js",
-    "./node-loader": "./esm/react-transport-dom-webpack-node-loader.js",
+    "./writer": {
+      "react-server": {
+        "node": "./writer.node.server.js",
+        "browser": "./writer.browser.server.js"
+      },
+      "default": "./writer.js"
+    },
+    "./writer.node.server": "./writer.node.server.js",
+    "./writer.browser.server": "./writer.browser.server.js",
+    "./node-loader": "./esm/react-server-dom-webpack-node-loader.js",
     "./node-register": "./node-register.js",
     "./package.json": "./package.json"
   },
   "browser": {
-    "./server.js": "./server.browser.js"
+    "./writer.js": "./writer.browser.server.js"
```

Comment:
What’s up with this change?

Reply:
I guess this is still a server even in the browser. So was the extension missing before?

Reply:
ah I see

>uses the .server.js convention to enforce that it's only loaded within a react-server environment.

Reply:
It’s weird that this convention wins and have to be one the end.

It’s also weird that since this reexports, the reexport is considered a hybrid. That just works because it doesn’t have dependencies on other server files.

We probably should change our CJS build output convention to include server at the end.